src/pack/rpm.go

```diff
@@ -0,0 +1,68 @@
+package pack
+
+import (
+	"fmt"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/tarantool/cartridge-cli/src/project"
+	"github.com/tarantool/cartridge-cli/src/rpm"
+)
+
+func packRpm(projectCtx *project.ProjectCtx) error {
+	// check context
+	if err := checkPackRpmCtx(projectCtx); err != nil {
+		// TODO: format internal error
+		panic(err)
+	}
+
+	// app dir
+	appDirPath := filepath.Join(projectCtx.PackageFilesDir, "/usr/share/tarantool/", projectCtx.Name)
+	if err := initAppDir(appDirPath, projectCtx); err != nil {
+		return err
+	}
+
+	// systemd dir
```

Comment:
I think a comments are redundant - the name `initSystemdDir `, `initTmpfilesDir ` say it.